S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+    if(lang=='turkey'or lang=='az' or lang=='tr'):      #converts uppercase to lowercase
+        for letter in word:
+            if(letter=='i'or letter=="I"):
+                I=u"\u0131"
+                print(I,end='')
+            else:
+                print(letter.lower(),end='')
+    
+    if(lang=='Greek' or lang=='el'):
+        leng=len(word)-1
```

Comment:
This is traditional way to iterate through the word, but python can iterate through letters of the word also. If it is needed you can optimise the code just by iterating through the words.